lib/media/gap_jumping_controller.js

```diff
@@ -482,6 +482,34 @@ shaka.media.StallDetector.MediaElementImplementation = class {
   constructor(mediaElement) {
     /** @private {!HTMLMediaElement} */
     this.mediaElement_ = mediaElement;
+    /**
+     * For listeners scoped to the lifetime of the instance.
+     * @private {shaka.util.EventManager}
+     */
+    this.audioEventManager_ = new shaka.util.EventManager();
```

Comment:
I might be missing something but I don't currently see any code that calls `.release()` on this `EventManager` object.